internal/e2e/check.js

```diff
@@ -9,7 +9,10 @@ function read(readPath) {
       .replace(/\r\n/g, '\n')
       .replace(/\\\\/g, '/')
       // In the sourcemap, CRLF is turned into a string encoding
-      .replace(/\\r\\n/g, '\\n');
+      .replace(/\\r\\n/g, '\\n')
+      // Replace platform specific directories in paths
+      .replace(/\/\w+-fastbuild\//g, '/__platform__-fastbuild/')
```

Comment:
is this related? could be a separate PR?

Reply:
I was getting different folder names outputted here compared to circleci. Were the golden files previously generated by circleci so they have the circleci platform? I figured this was better then me manually editing the golden files...